README.md

```diff
@@ -2,14 +2,20 @@
 Services: active-directory
 platforms: Android
 author: danieldobalian
+level: 100
+client: Android Mobile App
+service: Microsoft Graph
+endpoint: AAD V2
 ---
-
 # MSAL Android Sample Microsoft Graph API Sample 
 
 | [Getting Started](https://apps.dev.microsoft.com/portal/register-app?appType=mobileAndDesktopApp&appTech=android)| [Library](https://github.com/AzureAD/microsoft-authentication-library-for-android) | [API Reference](http://javadoc.io/doc/com.microsoft.identity.client/msal) | [Support](README.md#community-help-and-support)
 | --- | --- | --- | --- |
 
-The MSAL Android preview gives your app the ability to begin using the [Microsoft Cloud](https://cloud.microsoft.com) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. This sample demonstrates all the normal lifecycles your application should experience, including:
+![Build Badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/500/badge)
+
```

Comment:
Personally, I'd add something about the scenario (describe what the user does), have a screen copy, and probably one of the topology pictures like in the .NET samples?

Reply:
I would replace the Getting Started with https://docs.microsoft.com/azure/active-directory/develop/guidedsetups/active-directory-android - because is non-authenticated. The current link sends people directly to the app creation experience and people may be confused